### PR TITLE
Refactor/stencil4

### PR DESCRIPTION
**Pull request by dpellier**
- version update to Stencil v4
- remove already deprecated support of `custom-elements-bundle`
- simplify components stencil config
- simplify components globals
    - only one `globals.ts` file which contains only import of needed components (for local dev and test runners)
    - an optionnal `dev.ts` can be used if needed as a dev sandbox to test complex interaction
    - remove ods logger initialisation
- simplify components tsconfig (no more .dev, tsconfig.json is by default the dev version)
- run build action in parallel when possible to fasten global build time (on mac M1, went from 515s to 441s)

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build#1480.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build#1480.1 ✘  
* 
  * build ✘

File: packages/components/spinner/stencil.config.ts [hunk 1]
```diff
@@ -1,30 +1,10 @@
 import { Config } from '@stencil/core';
```
Comment (aesteves60) [inline packages/components/spinner/stencil.config.ts:1]: Type or remove because the type is implicit

**Review by aesteves60 — CHANGES_REQUESTED** [review]

**Comment by ovh-cds** [comment]
CDS Report build#1480.2 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build#1485.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build#1487.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build#1489.0 ✘  
* 
  * build ✘

**Comment by ovh-cds** [comment]
CDS Report build#1490.0 ✘  
* 
  * build ✘

File: packages/components/breadcrumb/stencil.config.ts [hunk 2]
```diff
@@ -1,6 +1,14 @@
 import { getStencilConfig } from '@ovhcloud/ods-common-stencil';
 import jestConfig from './jest.config';
 
+console.log(getStencilConfig({
```
Comment (aesteves60) [inline packages/components/breadcrumb/stencil.config.ts:4]: oups

**Review by aesteves60 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**